tests/pytests/test_cassis_drivers.py

```diff
@@ -0,0 +1,292 @@
+    "Quaternions": [
+      [-0.38852078202718,
+       -0.53534661398878,
+       -0.74986993015978,
+       0.012275694110544
+      ]
+    ],
+    "AngularVelocity": [
+      [-5.18973909108511e-04,
+       -2.78782123621868e-04,
+       2.84462861654798e-04
+      ]
+    ],
+    "ConstantRotation": [
+        0.0021039880161896, 
+        -5.08910327554815e-04,
+        0.99999765711961, 
+        0.98482103650022, 
+        0.17356149021485,
+        -0.0019837290716917, 
+        -0.17356007404082,
+        0.98482290292452, 
+        8.66356891752243e-04
+    ]
+  },
+  "BodyRotation": {
+    "TimeDependentFrames": [10014, 1],
+    "CkTableStartTime": 533471602.76595,
+    "CkTableEndTime": 533471602.76595,
+    "CkTableOriginalSize": 1,
+    "EphemerisTimes": [533471602.76595],
+    "Quaternions": [
+      [-0.84364286886959,
+       0.083472339272581,
+       0.30718999789287,
+       -0.43235793455935
+      ]
+    ],
+    "AngularVelocity": [
+        [3.16231952619494e-05,
+         -2.8811745785852e-05,
+         5.6516725802331e-05
+      ]
+    ]
+  },
+  "InstrumentPosition": {
+    "SpkTableStartTime": 533471602.76595,
+    "SpkTableEndTime": 533471602.76595,
+    "SpkTableOriginalSize": 1,
+    "EphemerisTimes": [533471602.76595],
+    "Positions": [
+      [-2707.0266303122,
+       2307.373459613,
+       2074.8887762465]
+    ],
+    "Velocities": [
+      [-1.6101681507607,
+       -4.1440662687653,
+       -0.48379032129765
+      ]
+    ]
+  },
+  "SunPosition": {
+    "SpkTableStartTime": 533471602.76595,
+    "SpkTableEndTime": 533471602.76595,
+    "SpkTableOriginalSize": 1,
+    "EphemerisTimes": [533471602.76595],
+    "Positions": [
+      [-206349081.4204,
+       17157784.002827,
+       13440194.20453
+      ]
+    ],
+    "Velocities": [
+      [-3.3908615862221,
+       -23.832213042409,
+       -10.839726759899
+      ]
+    ]
+  }
+}
+
+
+@pytest.fixture()
+def test_kernels(scope="module"):
+    kernels = get_image_kernels("CAS-MCO-2016-11-26T22.32.14.582-RED-01000-B1")
+    updated_kernels, binary_kernels = convert_kernels(kernels)
+    yield updated_kernels
+    for kern in binary_kernels:
+        os.remove(kern)
+
+def test_cassis_load(test_kernels, isis_compare_dict):
+    label_file = get_image_label("CAS-MCO-2016-11-26T22.32.14.582-RED-01000-B1", "isis")
+    isis_isd = ale.load(label_file, props={'kernels': test_kernels}, formatter="isis")
+
+    assert compare_dicts(isis_isd, isis_compare_dict) == []
+
+# ========= Test cassis ISIS label and naifspice driver =========
+class test_cassis_isis_naif(unittest.TestCase):
+    
+    def setUp(self):
+      label = get_image_label("CAS-MCO-2016-11-26T22.32.14.582-RED-01000-B1", "isis")
+      self.driver = TGOCassisIsisLabelNaifSpiceDriver(label)
+    
+    def test_short_mission_name(self):
+      assert self.driver.short_mission_name == "tgo"
+    
+    def test_exposure_duration(self):
+      assert self.driver.exposure_duration == 1.920e-003
+
```

Comment:
I'm not sure that these are very useful; would be happy to remove.

Reply:
These should test the functions implemented in the driver: instrument_id and ephemeris_start_time